internal/browsers.py

```diff
@@ -93,9 +93,6 @@ def get_browser(self, name, job):
             job['browser_info'] = self.browsers[name]
             if 'type' in self.browsers[name] and self.browsers[name]['type'] == 'Firefox':
                 if self.options.webdriver:
```

Comment:
No need for the "webdriver" option anymore - it can be removed since it is always webdriver now.